source/_posts/2017-07-07-you-may-not-be-a-polyglot-but-your-code-needs-to-be.markdown

```diff
@@ -0,0 +1,120 @@
+---
+layout: post
+title: "You May Not Be a Polyglot, but Your Code Needs to Be"
+author: Denisse Osorio de Large
+date: 2017-07-07
+tags: [engineering, polyglot, localization, internationalization, grace, hopper]
+---
+
+Last October, I had the privilege to speak at Grace Hopper, the world's largest gathering of women technologists. It was impressive to see 15,000 fellow female technologists gather together to share their experiences and technical expertise. To look into the audience, and see a room full of other female engineers was a great experience! 
+
+My talk was titled, You May Not Be a Polyglot, but Your Code Needs to Be.
```

Comment:
Seems like we could include a reference at the end to her talk that was given at Midwest.io:
https://www.youtube.com/watch?v=923ac4CSqsw